packages/nox/src-epetra/NOX_Epetra_BroydenOperator.C

```diff
@@ -759,17 +768,45 @@ BroydenOperator::replaceBroydenMatrixValues( const Epetra_CrsMatrix & mat)
     if( ierr )
     {
       std::cout << "ERROR (" << ierr << ") : "
-           << "NOX::Epetra::BroydenOperator::replaceBroydenMatrixValues(...)"
-           << " - Extract or Replace values error for row --> "
-           << row << std::endl;
+        << "NOX::Epetra::BroydenOperator::replaceBroydenMatrixValues(...)"
+        << " - Extract or Replace values error for row --> "
+        << row << std::endl;
+      throw "NOX Broyden Operator Error";
+    }
+  }
+}
+#else
+  void
+BroydenOperator::replaceBroydenMatrixValues( const Epetra_CrsMatrix & mat)
```

Comment:
Not a part of this PR, but the Broyden class needs to be rewritten to Thyra, not hard coded to Epetra.